praw/models/reddit/subreddit.py

```diff
@@ -480,7 +480,7 @@ def subscribe(self, other_subreddits=None):
             list of subreddits.
 
         """
-        data = {'action': 'sub', 'skip_inital_defaults': True,
```

Comment:
After taking the time to think about this, I think we can just fix the typo and be done with it as it technically was a bug on PRAW's end. The reason I added that there in the first place was I didn't like the fact that the result of the call is different depending on if the account was new or not.

Would you like to make a new PR for that fix?